data-format/message-header.md

```diff
@@ -16,13 +16,13 @@ See [Message Version](#message.md#version).
 
 ## Overview
 
-The message header is a component of the [message](#message.md).
+The message header is a component of the [message](message.md).
 
 The purpose of the message header is to define the authenticated metadata required for decryption of the [message body](#message-body), including:
 
-- the format of the [message](#message.md) and [message body](#message-body.md)
-- the encrypted data keys needed for decryption of the [message body encrypted content](#message-body.md#encrypted-content)
-- the [encryption context](#encryption-context.md)
+- the format of the [message](message.md) and [message body](message-body.md)
+- the encrypted data keys needed for decryption of the [message body encrypted content](message-body.md#encrypted-content)
+- the [encryption context](../framework/structures.md#encryption-context)
 
 ## Definitions
 
```

Comment:
Out of range again, but the "supported algorithm suites" link in the table should point to `../framework/algorithm-suites.md#supported-algorithm-suites`.